impls/src/adapters/mod.rs

```diff
@@ -59,64 +58,3 @@ pub trait SlateGetter {
 	/// Returns (Slate, whether it was in binary form)
 	fn get_tx(&self) -> Result<(Slate, bool), Error>;
 }
-
-/// select a SlateSender based on method and dest fields from, e.g., SendArgs
-pub fn create_sender(
-	method: &str,
-	dest: &str,
-	tor_config: Option<TorConfig>,
-) -> Result<Box<dyn SlateSender>, Error> {
-	let invalid = || {
-		ErrorKind::WalletComms(format!(
-			"Invalid wallet comm type and destination. method: {}, dest: {}",
-			method, dest
-		))
-	};
-
-	let mut method = method;
-
-	// will test if this is a tor address and fill out
-	// the http://[].onion if missing
-	let dest = match complete_tor_address(dest) {
-		Ok(d) => {
-			method = "tor";
-			d
-		}
-		Err(_) => dest.into(),
-	};
-
-	Ok(match method {
-		"http" => Box::new(HttpSlateSender::new(&dest).map_err(|_| invalid())?),
-		"tor" => match tor_config {
-			None => {
-				return Err(
-					ErrorKind::WalletComms("Tor Configuration required".to_string()).into(),
-				);
-			}
-			Some(tc) => Box::new(
-				HttpSlateSender::with_socks_proxy(&dest, &tc.socks_proxy_addr, &tc.send_config_dir)
-					.map_err(|_| invalid())?,
-			),
-		},
-		"keybase" => Box::new(KeybaseChannel::new(dest)?),
```

Comment:
With this change, is there any reason to import or keep around any of the keybase logic in adapters?

Reply:
Afaik we still support Keybase as a method. This particular block of code was just unused so I got rid of it entirely.